src/data/UserD2Repository.ts

```diff
@@ -196,4 +196,59 @@ export class UserD2Repository implements UserRepository {
             };
         });
     }
+
+    async getFromGroup(values: string[]): Async<User[]> {
+        const response = await this.api.metadata
+            .get({
+                userGroups: {
+                    fields: {
+                        id: true,
+                        users: true,
+                    },
+                    filter: {
+                        identifiable: {
+                            in: values,
+                        },
+                    },
+                },
+            })
+            .getData();
+
+        const userGroup = response.userGroups[0];
```

Comment:
Since the goal is to get an array of users from a group, can I rename the method to `getFromGroupByIdentifiable(value: Identifiable): Async<User[]>` and keep the same logic?

Reply:
Yes, that would work